### PR TITLE
Add MOFED RHCOS precompiled instructions

### PR DESCRIPTION
**Pull request by rollandf**
- Add steps for MOFED RHCOS precompiled instructions
- Update entrypoint.sh

### CODE REVIEW

**Comment by rollandf** [comment]
@krembu

**Comment by noama-nv** [comment]
@rollandf LGTM

**Review by adrianchiris — APPROVED** [review]

File: docs/advanced-configurations.rst [hunk 1]
```diff
@@ -335,6 +335,13 @@ RHEL
 
 Replace `RH_USERNAME` and `RH_PASSWORD` with your Red Hat account username and password.
 
+~~~~~
```
Comment (adrianchiris) [inline docs/advanced-configurations.rst:338]: why are we using `~~~`

i didnt find it in [1]

[1] https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Comment (rollandf) [reply docs/advanced-configurations.rst:338]: It seems to render OK:
https://mellanox.github.io/network-operator-docs/advanced-configurations.html#rhel-specific-build-parameters

**Pull request merged**